x11-themes/faenza-icon-theme/faenza-icon-theme-1.3.1-r1.ebuild

```diff
@@ -1,30 +1,28 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit gnome2-utils
+EAPI=7
+
+inherit xdg
 
 MY_P=${PN}_${PV}
 
 DESCRIPTION="A scalable icon theme called Faenza"
 HOMEPAGE="http://tiheum.deviantart.com/art/Faenza-Icons-173323228"
```

Comment:
https for HOMEPAGE 
( https://www.deviantart.com/tiheum/art/Faenza-Icons-173323228 )

Reply:
done